packages/fxa-admin-panel/src/components/PageAccountSearch/Account/index.tsx

```diff
@@ -146,6 +147,13 @@ export const Account = ({
             children={`${createdAtDate} (${createdAt})`}
             testId="account-created-at"
           />
+          <TableRowYHeader
+            header="Password Set"
+            children={`${
+              verifierSetAt != null && verifierSetAt! > 0 ? 'Yes' : 'No'
```

Comment:
I think you can remove the interpolation (just `children={ verifierSetAt...}` and the `!` in `verifierSetAt!`.